sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -364,7 +364,9 @@ export class EventHubSender extends LinkEntity {
     }
     return new Promise<number>(async (resolve, reject) => {
       const rejectOnAbort = () => {
-        const desc: string = `[${this._context.connectionId}] The create batch operation has been cancelled by the user.`;
+        const desc: string = `[${
+          this._context.connectionId
+        }] The create batch operation has been cancelled by the user.`;
```

Comment:
Did something change with the formatter settings again?

Reply:
Don't think so - formatting changes are best fixed early on, so that it doesn't show up on subsequent work done on this file by others.

Reply:
Yes, these are best fixed early on, but I doubt that this change is expected.

The last [3 commits to this file](https://github.com/Azure/azure-sdk-for-js/commits/dd9fcda6c1490a076b489025a6e1022544639307/sdk/eventhub/event-hubs/src/eventHubSender.ts) has been through your PRs. If none of those PRs had this change and the current PR does, it tells me that the repo set up from which this PR was created is using different settings.

Reply:
Yes I used to undo the changes because I would be asked to. :) 
But this likely needs to go in. Could you check if any new formatting changes appear on your side on saving?

Reply:
We should think about adding the `check-format` script to our CI so that it validates the code is formatted correctly. I usually try to run the `format` script before each commit and it sometimes catch things if the vscode extension didn't fix an issue on save.

Reply:
Agree, that would be super useful actually. I just ran the command and about 4 files were modified, none touched by this PR as such.